umbraco-marketplace-enterspeed.source.umbracocms.v11.json

```diff
@@ -0,0 +1,46 @@
+﻿{
```

Comment:
The marketplace files are automatically picked up by Umbraco when a filename in the root of the default branch matches the NuGet package name. (https://marketplace.umbraco.com/listing)

So if our new .net core package gets the name of Enterspeed.Source.UmbracoCms, we should add a file called umbraco-marketplace-enterspeed-source-umbracocms.json and we properbly don't need the v11 version of the file. 

If I'm allowed to push to your branch I could also do it. There might be some other changes, like I don't think we would have the v9 and v10 packages as related packages as they would be deprecated.